content/issues/3/llm-limit-case/index.md

```diff
@@ -46,15 +46,15 @@ In terms of text analysis tools more specifically, I've followed with interest [
 
 |                                                                                                               | "Data for good" | Data for co-liberation |
 |:------------------------------------------------------------------------------------------------------------- |:---------------:|:----------------------:|
-| Leadership by members of minoritized groups working in community                                              |                 |           x            |
-| Money and resources managed by members of minoritized groups                                                  |                 |           x            |
-| Data owned and governed by the community                                                                      |                 |           x            |
-| Quantitative data analysis “ground truthed” through a participatory, community-centered data analysis process |                 |           x            |
-| Data scientists are not rock stars and wizards, but rather facilitators and guides                            |                 |           x            |
-| Data education and knowledge transfer are part of the project design                                          |                 |           x            |
-| Building social infrastructure—community solidarity and shared understanding—is part of the project design    |                 |           x            |
-
-*Above: “Features of ‘data for good’ versus data for co-liberation, from Catherine D’Ignazio and Lauren F. Klein,* Data Feminism *(MIT Press, 2020), p. 140.*
+| Leadership by members of minoritized groups working in community                                              |                 |           ✓            |
+| Money and resources managed by members of minoritized groups                                                  |                 |           ✓            |
+| Data owned and governed by the community                                                                      |                 |           ✓            |
+| Quantitative data analysis “ground truthed” through a participatory, community-centered data analysis process |                 |           ✓            |
+| Data scientists are not rock stars and wizards, but rather facilitators and guides                            |                 |           ✓            |
+| Data education and knowledge transfer are part of the project design                                          |                 |           ✓            |
+| Building social infrastructure—community solidarity and shared understanding—is part of the project design    |                 |           ✓            |
+
+<table><caption>Above: Features of ‘data for good’ versus data for co-liberation, from Catherine D’Ignazio and Lauren F. Klein, Data Feminism (MIT Press, 2020), p. 140.</caption></table>
```

Comment:
This isn't semantic — it creates a separate, empty table with a caption.

As far as I can tell, there isn't a standard markdown format for table captions, and the format I found doesn't seem to be supported by goldmark.  It looks like doing this properly will require a shortcode — maybe we can adapt from this: https://willschenk.com/articles/2020/styling_tables_with_hugo/